doc/how_tos/custom_checkers.rst

```diff
@@ -117,14 +117,14 @@ Next we'll track when we enter and leave a function.
 
 .. code-block:: python
 
-  def __init__(self, linter=None):
+  def __init__(self, linter: PyLinter =None) -> None:
```

Comment:
Could you add the import for `PyLinter` in L42 -> `from pylint.lint import PyLinter`?